stratum/glue/status/posix_error_space.cc

```diff
@@ -2,16 +2,148 @@
 // Copyright 2018-present Open Networking Foundation
 // SPDX-License-Identifier: Apache-2.0
 
-
 #include "stratum/glue/status/posix_error_space.h"
 
 #include <errno.h>
 #include <stddef.h>
 #include <stdio.h>
+
 #include <string>
 
 #include "stratum/glue/logging.h"
 
+// Fake missing error codes on MacOS. Note, that they still have to be unique
+// and must not overlap with existing ones.
+#ifdef __APPLE__
+#ifndef ENOMEDIUM
+#define ENOMEDIUM (ELAST + __COUNTER__)
+#endif
```

Comment:
Since macros can't use preprocessor statements themselves, this block is written out by hand.